### PR TITLE
Bump rive-ios to v6.2, rive-android to 9.9.5, and remove iOS Skia support

### PR DESCRIPTION
**Pull request by lancesnider**
Breaking change: rive-react-native no longer supports the Skia renderer for iOS. 

## Testing Examples: 

1. Run `yarn bootstrap`
2. Run `yarn example start`
3. When Expo is ready, click `i` to start an iOS simulator 

You might have an issue where Expo tries to run on a previously used device, rather than a Simulator. If that happens, run `yarn example ios --simulator="iPhone 15"` (or whatever device). 

## Testing with a new Expo project: 

1. In `rive-react-native` run `npm pack` which will create `rive-react-native-8.0.0.tgz`
2. Create a new expo project and prebuild (formerly eject) it: https://docs.expo.dev/
4. In `Podfile` set the minimum platform to 14: `platform :ios, podfile_properties['ios.deploymentTarget'] || '14.0'`
5. Add this to your package.json dependencies: `"rive-react-native": "file:../rive-react-native/rive-react-native-8.0.0.tgz"` (adjust the path if needed) and run `npm i`
6. Add a Rive component `index.tsx` and run `npm run start`: 
```
import Rive, { Alignment, Fit, RiveRef } from 'rive-react-native'
...
<Rive
    url='https://cdn.rive.app/animations/vehicles.riv'
    fit={Fit.Contain}
    alignment={Alignment.Center}
    artboardName={'Truck'}
    style={{ width: 200, height: 200 }}
    autoplay={true}
  />
```

### CODE REVIEW

**Comment by HayesGordon** [comment]
Awesome, @lancesnider. We should bump Android as well. Due to recent changes, I think some native code will also need to be updated. I can do that, or if you're interested, we can hop on a call, and I'll show you.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -107,7 +107,7 @@ When an underlying runtime like [rive-ios](https://github.com/rive-app/rive-ios)
 Before running `yarn` at the root level of the project, set the appropriate version of the iOS runtime in `rive-react-native.podspec` at the root of the folder:
 
 ```
-s.dependency "RiveRuntime", "5.1.6"
+s.dependency "RiveRuntime", "6.2.0"
```
Comment (dskuza) [inline CONTRIBUTING.md:110]: We now have v6.2.1 which fixes some potentials for crashes on iOS, might want to bump the bump!

File: .gitignore [hunk 2]
```diff
@@ -47,6 +47,9 @@ npm-debug.log
 yarn-debug.log
 yarn-error.log
 
+# Ignore files generated by npm pack
+/*.tgz
+
```
Comment (lancesnider) [inline .gitignore:52]: Not totally related to this PR, but I use `npm pack` a lot to test this package in other repos and I don't want to accidentally commit the .tgz.

File: CONTRIBUTING.md (hunk 1 above)
Comment (lancesnider) [reply CONTRIBUTING.md]: Thanks. Added the newer version. I also make this section of the docs generic (`x.x.x`) so we don't have to keep updating it every time.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -107,7 +107,7 @@ When an underlying runtime like [rive-ios](https://github.com/rive-app/rive-ios)
 Before running `yarn` at the root level of the project, set the appropriate version of the iOS runtime in `rive-react-native.podspec` at the root of the folder:
 
 ```
-s.dependency "RiveRuntime", "5.1.6"
+s.dependency "RiveRuntime", "x.x.x"
```
Comment (HayesGordon) [inline CONTRIBUTING.md:110]: nice

**Review by HayesGordon — APPROVED** [review]
LGTM! Thanks for getting this in!

**Pull request merged**